atd-vze/src/views/Crashes/crashDataMap.js

```diff
@@ -1,104 +1,328 @@
-const crashDataMap = [
+export const crashDataMap = [
   {
     title: "Details",
     fields: {
-      case_id: "Case ID",
-      crash_date: "Crash Date",
-      crash_id: "Crash ID",
-      crash_speed_limit: "Speed Limit",
-      crash_time: "Crash Time",
-      day_of_week: "Day of Week",
-      fhe_collsn_id: "Manner of Collision ID",
-      last_update: "Last Updated",
-      investigator_narrative: "Investigator Narrative",
-      light_cond_id: "Light Condition",
-      obj_struck_id: "Object Struck ID",
-      road_type_id: "Roadway Type ID",
-      traffic_cntl_id: "Traffic Control ID",
-      wthr_cond_id: "Weather Condition ID",
+      case_id: {
+        label: "Case ID",
+        editable: false,
+      },
+      crash_date: {
+        label: "Crash Date",
+        editable: false,
+      },
+      crash_id: {
+        label: "Crash ID",
+        editable: false,
+      },
+      crash_speed_limit: {
+        label: "Speed Limit",
+        editable: false,
+      },
+      crash_time: {
+        label: "Crash Time",
+        editable: false,
+      },
+      day_of_week: {
+        label: "Day of Week",
+        editable: false,
+      },
+      fhe_collsn_id: {
+        label: "Manner of Collision ID",
+        editable: true,
+        uiType: "select",
```

Comment:
I like this pattern a lot - especially setting the uiType here and then handling it in the component. 🙌